include/ignition/gui/qml/PluginMenu.qml

```diff
@@ -38,7 +38,15 @@ Popup {
       Material.color(Material.Grey, Material.Shade200):
       Material.color(Material.Grey, Material.Shade900);
 
-  onOpened: searchField.forceActiveFocus()
+  property bool pluginAdded: false
+
+  onOpened: {
+    searchField.forceActiveFocus()
+    if (pluginAdded) {
+      searchField.selectAll()
+      pluginAdded = false;
```

Comment:
IMO I think we should always have it highlighted when the menu is opened. If users want to continue typing where they left off they can use the :arrow_right: key

Reply:
Changed back in aaa49a59